### ISSUE TITLE
Screen Reader Compatability

### ISSUE THREAD

**Issue by niapoor** [niapoor/Mailbox#1]
# Screen Reader Accessibility
### This game is currently not accessible to screen readers. While most text is in place to support this feature, the content describing each email is still missing. This is distinct from the email annotations that indicate whether they are scams or legitimate.
